.prettierignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-build
\ No newline at end of file
+build
+packages/cli/oclif.manifest.json
```

Comment:
No idea what this does, but as long as you expect it to be here, then it's good with me.

Reply:
Actually, if it's meant to be gitignored, then should we delete the one that's in the repo, too?

Reply:
it's not git ignored. it's prettier ignored

Reply:
Ah thanks haha

Reply:
it's not git ignored. it's prettier ignored.

It's a generated cli command manifest .. which I don't know where it is used but it needs to be there.